odoo/fields.py

```diff
@@ -3300,6 +3300,8 @@ def write(self, records, value):
                     property_definition.pop('value', None)
                 container[self.definition_record_field] = properties_definition
 
+                _logger.info('User #%i change definition of %r', records.env.user.id, container)
```

Comment:
Shouldn't we try to log changes also ? As otherwise we have no way to know a bit what was changed ?